### PR TITLE
Log initialized data sources

### PR DESCRIPTION
**Pull request by nichtich**
Show which data source has been initialized and how many are pending. The message may be extended with the number of triples.

### CODE REVIEW

**Comment by RubenVerborgh** [comment]
The idea is fine with me, but we might simply want to do this in `startWhenReady` (and rename that function then).

**Comment by nichtich** [comment]
The message should only be emitted on success (`initialized`), so it does not fit in `startWhenReady`.

**Comment by RubenVerborgh** [comment]
Fair enough!

File: bin/ldf-server [hunk 1]
```diff
@@ -169,9 +169,13 @@ else {
   // Create server, and start it when all data sources are ready
   var server = new LinkedDataFragmentsServer(config),
       pending = _.size(datasources);
-  _.each(datasources, function (settings) {
+  Object.keys(datasources).forEach(function (datasourceName) {
+    var settings = datasources[datasourceName]
```
Comment (rubensworks) [inline bin/ldf-server]: Another missing semicolon.

**Pull request closed without merging**